src/kernel/arch/x86/arch.zig

```diff
@@ -267,10 +267,13 @@ fn writeSerialCom1(byte: u8) void {
 ///
 /// Initialise serial communication using port COM1 and construct a Serial instance
 ///
+/// Arguments:
+///     IN payload: arch.BootPayload - The payload passed at boot. Not currently used by x86
+///
 /// Return: serial.Serial
 ///     The Serial instance constructed with the function used to write bytes
 ///
-pub fn initSerial() Serial {
+pub fn initSerial(payload: BootPayload) Serial {
```

Comment:
Maybe could have the name the same in kamin: `boot_payload`

Reply:
You missed the other 2 files :P

Reply:
Oh I didn't know there were more lines to change